server/src/main/java/org/jboss/as/server/deployment/module/DriverDependenciesProcessor.java

```diff
@@ -39,7 +38,7 @@
  */
 public class DriverDependenciesProcessor implements DeploymentUnitProcessor {
     private static final String SERVICE_FILE_NAME = "META-INF/services/java.sql.Driver";
-    private static final ModuleIdentifier JTA = ModuleIdentifier.create("javax.transaction.api");
+    private static final String JTA = "jakarta.transaction.api";
```

Comment:
Thanks for the review, I left the others as they are because they were public, but after reading your comment I realized the class is package visible, so they can also be replaced from ModuleIdentifier to String.